services/src/api/model/datatypes.rs

```diff
@@ -992,11 +991,31 @@ impl From<QueryRectangle<SpatialPartition2D>>
             spatial_bounds: value.spatial_bounds.into(),
             time_interval: value.time_interval.into(),
             spatial_resolution: value.spatial_resolution.into(),
-            attributes: BandSelection::first(), // TODO: adjust once API supports attribute selection
+            attributes: geoengine_datatypes::primitives::BandSelection::first(), // TODO: adjust once API supports attribute selection
```

Comment:
try_into and remove comment?

Reply:
no, the comment is about the fact, that the `QueryRectangle` in the API does not have the `attributes` yet, because they are not yet used by clients. That's why it always select the first band here instead.